app/__init__.py

```diff
@@ -8,10 +9,77 @@
 
 
 def create_app(config_name):
+    from app.models import BucketList
```

Comment:
This import was placed here to avoid the errors of circular imports. When it is moved up, errors are incurred.